dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/java_gateway.py

```diff
@@ -63,3 +63,130 @@ def gateway_result_checker(
     ):
         raise PyDSJavaGatewayException("Get result state not success.")
     return result
+
+
+java_gateway = launch_gateway()
```

Comment:
Function `launch_gateway` supported custom parameters in L30 -L34, should we consider about this?